source/nodejs/adaptivecards-designer/src/adaptivecards-designer.ts

```diff
@@ -9,12 +9,9 @@ export * from "./containers/index";
 export const defaultMicrosoftHosts: containers.HostContainer[] = [
     new containers.WebChatContainer("Bot Framework WebChat", "containers/webchat-container.css"),
     new containers.OutlookContainer("Outlook Actionable Messages", "containers/outlook-container.css"),
-    new containers.DarkTeamsContainer("Microsoft Teams - Dark", "containers/teams-container-dark.css"),
-    new containers.LightTeamsContainer("Microsoft Teams - Light", "containers/teams-container-light.css"),
-    new containers.DarkVivaConnectionsContainer("Viva Connections - Dark", "containers/viva-connections-container-dark.css"),
-    new containers.LightVivaConnectionsContainer("Viva Connections - Light", "containers/viva-connections-container-light.css"),
-    new containers.DarkCortanaContainer("Cortana Skills - Dark", "containers/cortana-container-dark.css"),
-    new containers.LightCortanaContainer("Cortana Skills - Light", "containers/cortana-container-light.css"),
+    new containers.TeamsContainer(containers.ColorTheme.Light),
+    new containers.VivaConnectionsContainer(containers.ColorTheme.Light),
+    new containers.CortanaContainer(containers.ColorTheme.Light),
```

Comment:
Likely you removed the dark theme

Reply:
Our containers default to light theme (we can change this if we want), and then the user can switch to dark theme using the drop down menu added in the previous PR.

Reply:
possible `new containers.CortanaContainer()` is enough.

Reply:
@licanhua I was thinking the same thing.
The only reason we would keep it is if we want consumers of the designer to have the option to create two separate containers instead of using the drop down menu.